edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/DockerConfig.cs

```diff
@@ -204,4 +237,17 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             public override bool CanConvert(Type objectType) => objectType == typeof(DockerConfig);
         }
     }
+
+    internal interface IEnvironmentWrapper
+    {
+        string GetVariable(string variableName);
+    }
+
+    internal class EnvironmentWrapper : IEnvironmentWrapper
+    {
+        public string GetVariable(string variableName)
```

Comment:
>string [](start = 15, length = 6)

for correctness this should prob return an Option<string>, then the caller can either use one of Option methods to get around having to null-check